task/07-yield-tasks.js

```diff
@@ -47,9 +63,19 @@ function* get99BottlesOfBeer() {
  *
  */
 function* getFibonacciSequence() {
-    throw new Error('Not implemented');
-}
+    let num = [0, 1];
+    let n;
 
+    for(let i = 0; i < num.length; i++) {
```

Comment:
this for-loop doesn't do anything useful. The while-loop below is infinite meaning `i` is always 0

Reply:
yes, you are right. thank you!